src/components/App.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component} from 'react';
+import { connect} from 'react-redux';
+
+import { addReminder, deleteReminder, clearReminders } from '../action';
+import moment from 'moment';
+import '../App.css';
+
+class App extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            text:'',
+            dueDate: ''
+        }
+    }
+
+    addReminder() {
+        console.log('this.state.dueDate', this.state.dueDate);
+        this.props.addReminder(this.state.text);
+    }
+
+    deleteReminder(id) {
+        this.props.deleteReminder(id);
+    }
+
+    
+    renderReminders() {
+        const { reminders } = this.props;
+        return (
+            <ul className='list-group col-sm-4'>
```

Comment:
Consider creating a new component for rendering these elements instead of cramming another method into this component.